src/docs/truffle/getting-started/debugging-your-contracts.md

```diff
@@ -94,6 +94,12 @@ $ truffle debug
 
 Regardless of how you start the debugger, once it is running you are not limited to debugging only the transaction you launched it with; it is possible to unload the current transaction and load a new one, as described below.
 
+NOTE: If your project was not compiled all at once (or under certain other
+conditions), the debugger will have to do its own compile of your project on
+startup.  This can be very slow.  If you compile your whole project at once,
+however, the debugger can likely avoid the initial recompile, speeding up
+startup greatly.
```

Comment:
Note on `NOTE:`: since this is getting rendered as hypertext, probably better to find an example use of one of those info boxes instead, so it can be in a blue or yellow box or whatever.

Reply:
Good idea!  Just pushed a revised version.